src/app/shared/components/toast/toast.component.ts

```diff
@@ -1,22 +1,26 @@
-import { Component, OnDestroy, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { Component, ElementRef, OnDestroy, OnInit, TemplateRef, ViewChild } from '@angular/core';
 import { NgbModal, NgbToast } from '@ng-bootstrap/ng-bootstrap';
 import { Toast } from '../../interfaces/toast';
 import { ToastService } from '../../services/toast.service';
 import { Subscription } from 'rxjs';
 import { NgFor, NgIf } from '@angular/common';
+import { ClipboardModule } from '@angular/cdk/clipboard';
 
 @Component({
   selector: 'app-toast',
   templateUrl: './toast.component.html',
   styleUrls: ['./toast.component.css'],
   standalone: true,
-  imports: [NgFor, NgIf, NgbToast],
+  imports: [NgFor, NgIf, NgbToast, ClipboardModule],
 })
 export class ToastComponent implements OnInit, OnDestroy {
   toastSubscription!: Subscription;
   selectedAlert!: Toast;
   @ViewChild('modal') modal!: TemplateRef<Element>;
+  @ViewChild('detailedErrorMessage') detailedErrorMessageElement!: ElementRef;
+  @ViewChild('CopyToClipboard') copyToClipboardElement!: ElementRef<HTMLButtonElement>;
```

Comment:
These are not used, forgot to remove them?

Reply:
Yes I forgot to remove them. These two are gone now.